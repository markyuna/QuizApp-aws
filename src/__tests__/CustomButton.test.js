// src/components/__tests__/CustomButton.test.js
import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import CustomButton from '../components/CustomButton/CustomButton';

it('renders correctly', () => {
  const tree = renderer.create(<CustomButton onPress={() => {}} text="Press me" />).toJSON();
  expect(tree).toMatchSnapshot();
});
