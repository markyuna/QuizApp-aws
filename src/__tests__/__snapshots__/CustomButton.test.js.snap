// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    [
      {
        "alignItems": "center",
        "borderRadius": 5,
        "marginVertical": 5,
        "padding": 15,
        "width": "100%",
      },
      {
        "backgroundColor": "#FFA726",
      },
      {},
    ]
  }
>
  <Text
    style={
      [
        {
          "color": "white",
          "fontWeight": "bold",
        },
        undefined,
        {},
      ]
    }
  >
    Press me
  </Text>
</View>
`;
